templates/registration/register.html

```diff
@@ -25,27 +22,17 @@ <h5 class="button-font"><i class="fa fa-google button-icon-padding" aria-hidden=
                     </div>
                 </a>
             </div>
-
-            <div class="button-container">
-                <a class="manual-font" data-open="signup-modal">
-                    <div class="sign-up-button manually">
-                        <h5 class="button-font"><i class="fa fa-envelope button-icon-padding" aria-hidden="true"></i>Sign Up with e-mail</h5>
-                    </div>
-                </a>
-            </div>
-            <div class="reveal center" id="signup-modal" data-reveal>
-                <form action="" method="POST">
-                    {% csrf_token %}
-                    <p class="color-grey">
-                        Username and password are case-sensitive
-                    </p>
-                    {{ form.as_p }}
-                    <input class="button submit-button" type="submit" value="Register" />
-                </form>
-                <button class="close-button" data-close aria-label="Close modal" type="button">
-                    <span aria-hidden="true">&times;</span>
-                </button>
-            </div>
+            <form class="small-form-input" action="" method="POST">
+                {% csrf_token %}
+                <p class="color-grey">
+                    Username and password are case-sensitive
+                </p>
+                {{ form.as_p }}
+                <p>
+                    I agree to the <a href="{% url 'terms' %}">Terms & Conditions</a>
```

Comment:
Should this be something like "By signing up, I agree..."?

Reply:
Yeah, I think that makes more sense

Reply:
Sorry, but after thinking about this more I think using the pronoun _I_ is unclear. Perhaps this should be something more like:
> By clicking Register, you agree to the Terms...